app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():

```

Comment:
It's good practice to handle potential `TypeError` exceptions that could be raised when comparing `None` with a date. If the 'expiration_date' key does not exist in the 'vaccine' dictionary, `visitor['vaccine'].get('expiration_date')` will return `None`, and comparing `None` with a date will raise a `TypeError`. You could handle this by adding a check for `None` before the comparison: `if expiration_date is None or expiration_date < datetime.date.today():`.